backend/src/helpers/auth.js

```diff
@@ -90,9 +91,39 @@ const verifyUserExists = (token) => {
         }
     });
 }
+
+/**
+ * Gets the User info based off the keycloak bearer token, and eventually the database information.
+ *
+ * @var {object}  req  The request object.
+ * @todo  Get user info from the database, and merge with return object.
+ * @todo  Add tests after logic becomes more stable.
+ *
+ * @returns {object}  The User object.
+ * 
+ */
+const getUserInfo = req => {
+    const token = getBearerTokenFromRequest(req)
+    const decodedToken = jwt.decode(token, { complete: true });
+    if (decodedToken) {
+        const payload = decodedToken.payload
+        // This role will eventually come from the database.
+        role = 'admin'
```

Comment:
Not sure how this didn't break. This should be:
```suggestion
        const role = 'admin';
```